cellprofiler/modules/tests/test_saveimages.py

```diff
@@ -656,7 +656,7 @@ def test_00_10_load_v10(self):
         pipeline = cpp.Pipeline()
         image_folder_text = pipeline.encode_txt(
             "%s|%s" % (cpprefs.ABSOLUTE_FOLDER_NAME, 
-                       cpp.utf16encode(cpmt.example_images_directory())))
+                       cpmt.example_images_directory().encode("utf-16")))
```

Comment:
Unfortunately, cpp.utf16encode wasn't named so correctly. I think this should be "utf-8", as inappropriate as that might seem. There are a lot of unit tests that inadvertently fail because of this and I hope a few that purposefully test for the encoding and fail.